app/models/recruitment_cycle.rb

```diff
@@ -1,5 +1,6 @@
 module RecruitmentCycle
   CYCLES = {
```

Comment:
Probably not for this PR, but is this `CYCLES` hash still really needed? It feels like we can run everything off `years_visible_to_providers` and turn this into a method like:
```
def good_method_name(year)
  "#{cycle_name(year)} (starts #{year})"
end
```

Reply:
I guess it may be useful to retain a list of all cycles for the support interface (all cycles vs. visible to providers). But we could make this more explicit with something like `years_visible_to_support`, along the lines of `years_visible_to_providers`, then implement a method for the description, as you suggested.